yt_dlp/extractor/tiktok.py

```diff
@@ -35,6 +35,13 @@ class TikTokBaseIE(InfoExtractor):
     _WEBPAGE_HOST = 'https://www.tiktok.com/'
     QUALITIES = ('360p', '540p', '720p', '1080p')
 
+    def _real_initialize(self):
+        api_hostname = traverse_obj(
+            self._configuration_arg('api_hostname', ie_key=TikTokIE.ie_key()), -1)
+        if api_hostname:
+            self.write_debug(f'Imported API hostname from extractor argument: {api_hostname}')
+            self._API_HOSTNAME = api_hostname
```

Comment:
```suggestion
    @property
    def _API_HOSTNAME(self):
        return self._configuration_arg(
            'api_hostname', ['api16-normal-c-useast1a.tiktokv.com'], ie_key=TikTokIE)[0]
```

Reply:
Should this be done for `DouyinIE` too? It redefines `_API_HOSTNAME` / would need a different default value, but I think we would want the extractor-arg to work for it as well

Reply:
We want the same extractor-arg to work for douyin too? Won't that cause one of these to be always broken whenever the arg is used?

Reply:
Oh, good point. We could do `ie_key=DouyinIE` for douyin, but I guess it's not necessary

Reply:
Yes, we can copy it to Douyin when need arises